packages/react/src/editor/styles.css

```diff
@@ -0,0 +1,426 @@
+/* Default theme params, overwritten by BlockNoteView */
```

Comment:
brainstorming:

- Can we have multiple editors (with different themes?)? Should we apply vars to root / should we clean them?
- Can / should  we specify a theme completely in CSS?
- Should we reuse "base" (color) variables?

fix ->
- clearer separation of layers / concerns, let's zoom out and first create pure CSS solution (without themes from code). then build on top of that